tsconfig.lib.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "target": "es2015",
+    "module": "es2015",
```

Comment:
`es2015` is a full synonym of `es6`. If you want a version of JavaScript that doesn't support `let` and `const`, you should use `es5`.

Reply:
Great point, thanks. It's easy to confuse es namings 😅